org.sil.Bloom.yml

```diff
@@ -0,0 +1,647 @@
+                      - make install
+                  cleanup:
+                      - "/include"
+                      - "*.a"
+                  sources:
+                      - type: git
+                        url: https://github.com/mirrorer/giflib.git
+                        commit: fa37672085ce4b3d62c51627ab3c8cf2dda8009a
+                        # Note that libgdiplus needs giflib to have GifQuantizeBuffer.
+                        # Some giflibs need adjustment to have that built in.
+          - name: xchm
+            # Help file viewer
+            cleanup:
+                - "/include"
+                - "*.a"
+                - "/share/icons"
+                - "/share/applications"
+            sources:
+                - type: git
+                  url: https://github.com/rzvncj/xCHM
+                  # Version 1.31 is used in Ubuntu 20.04.
+                  tag: "1.31"
+                  commit: e983d11fd546c961501f736b2c6f93178c92fa97
+                - type: patch
+                  # xchm would otherwise need to write to files ~/.xchmXXXXXX and
+                  # ~/.xchm. This won't be needed after upgrading to a newer xchm
+                  # version and wxWidgets v3.1.1+.
+                  path: patches/xchm/no-config.patch
+            modules:
+                - name: chmlib
+                  cleanup:
+                      - "/include"
+                      - "*.a"
+                  sources:
+                      - type: git
+                        url: https://github.com/jedwing/CHMLib
+                        # Latest commit, from 2009-07-03.
+                        commit: 2bef8d063ec7d88a8de6fd9f0513ea42ac0fa21f
+                      - type: script
+                        commands:
+                            - autoreconf --install
+                - name: wxWidgets
+                  cleanup:
+                      - "/include"
+                      - "/lib/wx/include"
+                      - "*.a"
+                  sources:
+                      - type: git
+                        url: https://github.com/wxWidgets/wxWidgets
+                        # v3.0.4 is used in Ubuntu 20.04.
+                        # v3.1.1+ is needed for xchm to write to XDG_CONFIG_HOME.
+                        # However, v3.1.0 appears to introduce a problem rendering the
+                        # chm html. v3.0.5.1 still worked, rendering the html.
+                        tag: v3.0.4
+                        commit: 721d62adde3f8ba8704a9cf56efeb050f652dfbf
+          - name: wget
+            buildsystem: autotools
+            cleanup:
+                # Don't ship.
+                - "*"
+            sources:
+                - type: archive
+                  url: https://ftp.gnu.org/gnu/wget/wget-1.20.3.tar.gz
+                  sha512: e8b82b40e270296228094a78d47f81580bdbdea9e6b93fd61b37dccb39430aeb9bda5397dc53a31c952a61629383c7e2a8c8abf414c8a4dd369af6ecf2717e6c
+          - name: dbus-glib
```

Comment:
this module is part of the shared-modules. you should use that instead.

https://github.com/flathub/shared-modules/

Reply:
done.